README.md

```diff
@@ -87,6 +87,34 @@ Then for each game node (typically one per CPU/core):
 PORT={port} SERVER={node-name} node server/gamenode
 ```
 
+### Running and Testing
+
+The game server should be accessible by browsing to localhost:4000. 
+
+You can register 2 or more users, to play against yourself. 
+They can have fake email addresses. 
+You can login as both users either from 2 different browsers, or by 
+using an incognito window. 
+
+There is no concept of an 'admin' user.
```

Comment:
This isn't true, but it's not necessary for development

Reply:
Reworded.